### ISSUE TITLE
[BUG] $excel->setCreator does not work for none-ascii characters

### ISSUE THREAD

**Issue by salarmehr** [PHPOffice/PHPExcel#876]
`$excel->setCreator('آرد')` (and other similar methods) set some random characters on file. I also tried encoding the string to uft16 and ucs without success.

**Comment by MarkBaker** [comment]
I'd hope that it would be `$excel->getProperties()->setCreator('آرد')` because the PHPExcel object doesn't have a `setCreator()` method

**Comment by salarmehr** [comment]
yes it is. actually I encounter the issue via the Laravel excel middle package.

**Comment by salarmehr** [comment]
@MarkBaker  So, do you confirm the issue? any workaround?

**Comment by lourdas** [comment]
Same here, I want to write some properties in Greek and I get characters like 'ÎšÏ„Î·Î½Î¿Ï„ÏÎ¿Ï†Î¹ÎºÏŒ ÎœÎ·Ï„ÏÏŽÎ¿ - Î•ÎºÎ¼ÎµÏ„Î±Î»Î»ÎµÏÏƒÎµÎ¹Ï‚ Î±Î½Î¬ Î .Î•.'.
